todo_app/data/ToDoItem.py

```diff
@@ -0,0 +1,49 @@
+_BASE_LIST_ID = '60a6c184fd2064526bdf740'
+_NOT_STARTED_LIST_ID = 'b'
+_BEING_DONE_LIST_ID = 'c'
+_COMPLETED_LIST_ID = 'd'
```

Comment:
These fields aren't sensitive but would be better extracted to an environment variable so that they can vary - we don't want development & production pointing to the same Trello board

In addition, I'd advise shifting to just matching list ids directly rather than with the final character - though that's often valid, there's no guarantee that will always be the case (e.g. if a user has recreated a list on a board, or even just if you use the final character and rotate 1y -> 1z -> 20 )

Reply:
Moved to environment variables in commit 4f9b126